granulate_utils/config_feeder/client/bigdata/dataproc.py

```diff
@@ -0,0 +1,40 @@
+import json
+import subprocess
+from typing import Any, Dict, Optional
+
+from granulate_utils.config_feeder.core.models.node import NodeInfo
+
+
+def get_dataproc_node_info() -> Optional[NodeInfo]:
```

Comment:
Please provide a link in a comment to where the attributes were taken from (documentation) if available, if not please specify on which dataproc versions it was tested

Reply:
Can you please also test it on dataproc 1.4?